File: Programmers/Lv. 2/기능개발.js
function solution(progresses, speeds) {
    let answer = [];
    
    const developAccesses = progresses.map((progress, idx)=> Math.ceil((100 - progress) / speeds[idx]));
    let developedDay = developAccesses[0]
    let developedDayCount = 0;
    
    developAccesses.forEach(developAccess => {
        if(developedDay >= developAccess){
            developedDayCount++;
        } else {
            developedDay = developAccess;
            answer.push(developedDayCount);
            developedDayCount = 1;
        }
    })

    if(developedDayCount) {
        answer.push(developedDayCount);
    }
    
    return answer;
}

File: Programmers/Lv. 2/기능개발.py
import math

def solution(progresses, speeds):
    answer = []
    stack = [math.ceil((100 - progress) / speed) for progress, speed in zip(progresses, speeds)]
        
    currentDeployDay = stack[0]
    deployCount = 1
    
    for endDay in stack[1:]:
        if currentDeployDay >= endDay:
            deployCount += 1
        else:
            answer.append(deployCount)
            currentDeployDay = endDay
            deployCount = 1
            
    answer.append(deployCount)
              
    return answer
